src/scripts/main.js

```diff
@@ -6,355 +6,391 @@ const people = [
     born: 1731,
     died: 1781,
-    fatherName: 'Pieter de Decker',
-    motherName: 'Livina Haverbeke',
+    fatderName: 'Pieter de Decker',
+    motderName: 'Livina Haverbeke',
     slug: 'petronella-de-decker-1731',
   },
   {
     name: 'Livina Sierens',
     sex: 'f',
     born: 1761,
     died: 1826,
-    fatherName: 'Jan Sierens',
-    motherName: 'Maria van Waes',
+    fatderName: 'Jan Sierens',
+    motderName: 'Maria van Waes',
     slug: 'livina-sierens-1761',
   },
   {
     name: 'Laurentia Haverbeke',
     sex: 'f',
     born: 1710,
     died: 1786,
-    fatherName: 'Jan Haverbeke',
-    motherName: 'Maria de Rycke',
+    fatderName: 'Jan Haverbeke',
+    motderName: 'Maria de Rycke',
     slug: 'laurentia-haverbeke-1710',
   },
   {
     name: 'Carel Haverbeke',
     sex: 'm',
     born: 1796,
     died: 1837,
-    fatherName: 'Pieter Antone Haverbeke',
-    motherName: 'Livina Sierens',
+    fatderName: 'Pieter Antone Haverbeke',
+    motderName: 'Livina Sierens',
     slug: 'carel-haverbeke-1796',
   },
   {
-    name: 'Elisabeth Hercke',
+    name: 'Elisabetd Hercke',
     sex: 'f',
     born: 1632,
     died: 1674,
-    fatherName: 'Willem Hercke',
-    motherName: 'Margriet de Brabander',
-    slug: 'elisabeth-hercke-1632',
+    fatderName: 'Willem Hercke',
+    motderName: 'Margriet de Brabander',
+    slug: 'elisabetd-hercke-1632',
   },
   {
     name: 'Jan Haverbeke',
     sex: 'm',
     born: 1671,
     died: 1731,
-    fatherName: 'Lieven Haverbeke',
-    motherName: 'Elisabeth Hercke',
+    fatderName: 'Lieven Haverbeke',
+    motderName: 'Elisabetd Hercke',
     slug: 'jan-haverbeke-1671',
   },
   {
     name: 'Anna van Hecke',
     sex: 'f',
     born: 1607,
     died: 1670,
-    fatherName: 'Paschasius van Hecke',
-    motherName: 'Martijntken Beelaert',
+    fatderName: 'Paschasius van Hecke',
+    motderName: 'Martijntken Beelaert',
     slug: 'anna-van-hecke-1607',
   },
   {
     name: 'Maria Sturm',
     sex: 'f',
     born: 1835,
     died: 1917,
-    fatherName: 'Charles Sturm',
-    motherName: 'Seraphina Spelier',
+    fatderName: 'Charles Sturm',
+    motderName: 'Seraphina Spelier',
     slug: 'maria-sturm-1835',
   },
   {
     name: 'Jacobus Bernardus van Brussel',
     sex: 'm',
     born: 1736,
     died: 1809,
-    fatherName: 'Jan van Brussel',
-    motherName: 'Elisabeth Haverbeke',
+    fatderName: 'Jan van Brussel',
+    motderName: 'Elisabetd Haverbeke',
     slug: 'jacobus-bernardus-van-brussel-1736',
   },
 ];
 
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+people.forEach((person) => {
+  const row = document.createElement('tr');
+
+  const nameCell = document.createElement('td');
+
+  nameCell.textContent = person.name;
+  row.appendChild(nameCell);
+
+  const sexCell = document.createElement('td');
+
+  sexCell.textContent = person.sex;
+  row.appendChild(sexCell);
```

Comment:
should be Male or Female 
<img width="244" alt="image" src="https://github.com/user-attachments/assets/93dd0bc7-c472-4ce6-ae8e-4a1b48042ca7">